lib/Dialect/SV/Transforms/GeneratorCallout.cpp

```diff
@@ -90,6 +132,8 @@ void HWGeneratorCalloutPass::processGenerator(
   // explicitly.
   generatorArgs.push_back("--moduleName");
   generatorArgs.push_back(moduleName);
+  generatorArgs.push_back("--hierarchichalName");
```

Comment:
A generator callout is suppose to be a simple module-like thing -> implementation of module-like thing.  It shouldn't be for exfiltrating global structure.